setup.cfg

```diff
@@ -53,6 +53,7 @@ addopts =
 filterwarnings =
     error
     ignore:module 'ssl' has no attribute 'OP_NO_COMPRESSION'. The Python interpreter is compiled against OpenSSL < 1.0.0. Ref. https.//docs.python.org/3/library/ssl.html#ssl.OP_NO_COMPRESSION:UserWarning
+    ignore:The loop argument is deprecated since Python 3.8, and scheduled for removal in Python 3.10.:DeprecationWarning:asyncio.base_events
```

Comment:
Could you add a comment with the explanation right above this line? Maybe with links too. And then, go ahead and merge it.

Reply:
Is it worth merging this when it doesn't fix all the tests? Or should we just merge #6006 instead? Or both?

I'll probably just open an issue though to remind myself to revert the changes in a few weeks.

Reply:
I think if it improves things at least a little, we should merge it. And yes, we could revert later. Plus I'd combine with #6006 too.

Reply:
Right, you'll need to tweak the required status to get them merged.